app/uk/gov/hmrc/tariffclassificationfrontend/models/ReviewStatus.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+object ReviewStatus extends Enumeration {
+  type ReviewStatus = Value
+  val IN_PROGRESS, UPHELD, OVERTURNED = Value
+
+  def format(status: Option[ReviewStatus]): String = {
+    status match {
+      case Some(IN_PROGRESS) => "Under review"
+      case Some(UPHELD) => "Review upheld"
+      case Some(OVERTURNED) => "Review overturned"
+      case None => "None"
+      case _ => ???
```

Comment:
best to avoid using `???`
replace with `Some(s)` => throw new IllegalArgumentException(....bla bla bla )

Reply:
see what I have done for the appeal status